solidity-v1/dashboard/src/css/progress-bar.less

```diff
@@ -22,6 +22,11 @@
     overflow: hidden;
     margin: 1.5rem 0;
   }
+
+  &--secondary-color {
+    border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+  }
```

Comment:
I think we don't need it because we refactored progress bar component. Please see https://github.com/keep-network/keep-core/commit/84df0563e90b72995d9c5b10d9751f8497d52f86 https://github.com/keep-network/keep-core/pull/2668/#discussion_r749105476

Reply:
Fixed in c7aefc8635c06536747aac6cab1afdc12f08572c